web/source/kmwnative.ts

```diff
@@ -53,7 +53,7 @@ if(!window['keyman']['initialized']) {
       if(s) {
         bg.pending=true;
         window.setTimeout(function() {
-            if(bg.pending) {
+            if(bg.pending && keymanweb.options.useAlerts) {
```

Comment:
Would prefer for this check to be on `wait` calls... but upon investigation... it appears that `wait` is actually internal only; we never actually documented it as API.  Huh.

So... maybe it's fine not to do the checks in the same way we did for `alert`, which is published API?

Reply:
Thoughts, @mcdurdin?  An initial impression based on the comment above is fine.

Reply:
LGTM